setup.py

```diff
@@ -29,7 +29,7 @@
         "requests>=2.18",
         "jsons==1.0.0",
         "numpy==1.19.4",
-        "microsoft-bonsai-api==0.1.2",
+        "microsoft-bonsai-api>=0.1.3",
```

Comment:
It's safer to lock down the version in case of future breaking changes. ==0.1.3 is preferable. If you tested it and it works I approve.